tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -70,6 +73,14 @@ private void WindowEx_Closed(object sender, WindowEventArgs args)
     {
         DBarWindow?.Close();
         _hotKeyHelper?.UnregisterHotKey();
+        _servicingHelper?.Unregister();
+    }
+
+    public void HandleSessionEnd()
+    {
+        App.Log("DevHome.PI_SessionEnd", LogLevel.Info);
+        DBarWindow?.Close();
+        Process.GetCurrentProcess().Kill();

```

Comment:
Can we call WindowEx_Closed here? So we don't have to kill the process.

Reply:
Terminate process is much more reliable. I imagine in the future we will need to have a more complex flow to ensure any in-progress work is saved.